tests/plotting/test_plotting.py

```diff
@@ -2448,15 +2448,39 @@ def test_write_gif(sphere, tmpdir):
     assert os.path.getsize(path)
 
 
-def test_ruler(sphere):
-    sphere = pyvista.Sphere()
+def test_ruler():
     plotter = pyvista.Plotter()
-    plotter.add_mesh(sphere)
+    plotter.add_mesh(pyvista.Sphere())
     plotter.add_ruler([-0.6, -0.6, 0], [0.6, -0.6, 0], font_size_factor=1.2)
     plotter.view_xy()
     plotter.show()
 
 
+def test_legend_scale(sphere):
+    plotter = pyvista.Plotter()
+    plotter.add_mesh(sphere)
+    plotter.add_legend_scale(color='red')
+    plotter.show()
+
+    plotter = pyvista.Plotter()
+    plotter.add_mesh(sphere)
+    plotter.add_legend_scale(color='red', xy_label_mode=True)
```

Comment:
I'm not sure how to address this, but the numbers are near indistinguishable from the default before.

Reply:
Yeah... I'm over working on this... I mainly just wanted to use the legend bar at the bottom, and this whole thing has tons of issues upstream and controlling the font is a headache. It'll be a rarely used feature and I'm fine with the support and testing not being optimal

Reply:
I agree that a regression here is a minor concern.  More of a comment in case it was a quick fix.